.ci-config/project.yml

```diff
@@ -0,0 +1,6 @@
+projects:
+  - name: diff-java-tool
+    scm: git
+    url: file:///home/runner/work/test-configs/test-configs/diff-java-tool
+    reference: main
```

Comment:
continuation of https://github.com/checkstyle/test-configs/pull/192#discussion_r1786905807

try:
```
    url: file:///home/runner/work/test-configs
    reference: main
```

as it most likely neded folder with ".github"

Reply:
done, waiting for CI